src/modules/Builder/components/Configuration/components/Wireless/Wireless.jsx

```diff
@@ -121,66 +52,91 @@ export default class Wireless extends Component {
         }
 
-        buttonStateCallback("wirelessPSK", PSKValid);
+        updateValidStatesAction({ PSKValid });
 
         return PSKValid;
-    }
-
-    render() {
-        const { SSID, PSK, enableWireless } = this.state;
-
-        return (
-            <Card>
-                <Card.Body>
-                    <div className="configuration-wireless">
-                        <Form.Group>
-                            <Form.Check
-                                type="checkbox"
-                                id="show-ordinary-password"
-                                label="Support Wi-Fi"
-                                onChange={this.onCheckboxChange}
-                                checked={enableWireless}
-                            />
-                        </Form.Group>
-                        <Form.Group>
-                            <Form.Label>Network name</Form.Label>
-                            <Input
-                                id="SSIDID"
-                                name="SSID"
-                                type="text"
-                                value={SSID}
-                                onChange={this.onChangeData}
-                                disabled={!enableWireless}
-                                validationFunc={this.SSIDValidor}
-                            />
-                            <div className="error invalid-feedback">
-                                Network name must be between 1 and 31 characters of Latin letters,
-                                numbers or a dot character
-                            </div>
-                        </Form.Group>
-                        <Form.Group>
-                            <Form.Label>Password</Form.Label>
-                            <Input
-                                id="PSKID"
-                                name="PSK"
-                                type="password"
-                                value={PSK}
-                                onChange={this.onChangeData}
-                                disabled={!enableWireless}
-                                validationFunc={this.PSKValidor}
-                            />
-                            <div className="error invalid-feedback">
-                                Password must be either empty or between 8 and 63 characters
-                                of Latin letters, numbers or a dot character
-                            </div>
-                        </Form.Group>
-                    </div>
-                </Card.Body>
-            </Card>
-        );
-    }
-}
+    };
+
+    return (
+        <Card>
+            <Card.Body>
+                <Form>
+                    <Form.Group>
+                        <Form.Check
+                            type="checkbox"
+                            id="show-ordinary-password"
+                            label="Support Wi-Fi"
+                            onChange={() => updateConfigurationParamsAction(
```

Comment:
Add `name` prop to `Form.Check` and pass field name and field value to `updateConfigurationParamsAction`.

Reply:
Form.Check not return name and value on onChange event

Reply:
Then take `name` and `value` from `event.target` object.

Reply:
Returned value is `on/off`, we need boolean variable

Reply:
I can use name, but not value